### PR TITLE
adding basic code to use gz files

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,40 +1,52 @@
...
 language: python
-sudo: required
-group: deprecated-2017Q4
+sudo: false
 env:
-  - PYTHON_VERSION=2.7
+  - PYTHON_VERSION=3.6
 before_install:
-  - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+  - wget http://repo.continuum.io/miniconda/Miniconda3-3.7.3-Linux-x86_64.sh -O miniconda.sh
```
Comment (ElDeveloper) [inline .travis.yml:7]: Any reason not to use the latest?

File: qp_target_gene/pick_otus.py [hunk 2]
```diff
@@ -55,6 +58,15 @@ def generate_pick_closed_reference_otus_cmd(filepaths, out_dir, parameters):
...
     seqs_fp = filepaths['preprocessed_fasta'][0]
 
+    if not test and is_zipfile(seqs_fp):
+        seqs_fp_fna = join(out_dir, 'seqs.fna')
+        cmd_ungz = 'gunzip -c %s > %s' % (seqs_fp, seqs_fp_fna)
+        std_out, std_err, return_value = system_call(cmd_ungz)
+        if return_value != 0:
+            error = ("Std out: %s\nStd err: %s\n\nCommand run was:\n%s"
+                     % (std_out, std_err, cmd_ungz))
+            raise RuntimeError(error)
```
Comment (ElDeveloper) [inline qp_target_gene/pick_otus.py:68]: I think we could remove this block if instead we prepend the gunzip command to the pick_closed__reference_otus.py command. What do you think? That way error handling, etc is handled all in the same place.

File: .travis.yml (hunk 1 above)
Comment (antgonza) [reply .travis.yml:7]: Nop, will change.

Comment (antgonza) [reply .travis.yml:7]: Well, looks like libcfortran is different between version; returned so tests pass

**Comment by ElDeveloper** [comment]
Thanks @antgonza

**Pull request merged**